src/table/src/requests.rs

```diff
@@ -105,4 +105,8 @@ pub struct DeleteRequest {
     ///
     /// The key is the column name, and the value is the column value.
     pub key_column_values: HashMap<String, VectorRef>,
+
+    pub catalog_name: String,
+    pub schema_name: String,
+    pub table_name: String,
```

Comment:
Users don't need to pass catalog/schema/table names. We could locate the table in the SQL handler.

Reply:
Now get the table in this way:
```rust
let table = self.get_table(&table_ref)?;
``` 
Do we have other methods to get table in the handler?

Reply:
You could pass the `QueryContext` to this method and convert the delete statement into the `table::DeleteRequest` inside `SqlHandler::delete()`.
```rust
pub(crate) async fn delete(&self, query_ctx: &QueryContext, stmt: Delete) -> Result<Output> {}
```

Reply:
I see what you mean!